app/views/browse/service-instance.html

```diff
@@ -67,26 +81,67 @@ <h1 class="contains-actions">
                           <dd ng-if-end>
                             {{serviceInstance | serviceInstanceConditionMessage}}
                           </dd>
-                          <dt ng-if-start="serviceClass.spec.description || serviceClass.spec.externalMetadata.longDescription">Description:</dt>
-                          <dd ng-if-end>
+                        </dl>
+                        <div class="hidden-lg">
+                          <h3 ng-if-start="serviceClass.spec.description || serviceClass.spec.externalMetadata.longDescription">Description</h3>
+                          <dl ng-if-end class="dl-horizontal left">
```

Comment:
A dl should only contain [dt, dd, or div/script/template elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).  So either don't use a dl or change the p elements to divs and add a margin using a helper class (e.g., mar-bottom-md)?  I think I prefer the former as that maintains the semantic value of the p.

Reply:
No need for the dl.  Just remove it and render the p as individual elements.